firestore/integration_test_internal/src/util/future_test_util.h

```diff
@@ -12,9 +12,11 @@
 namespace firebase {
 
 // Prints a human-friendly representation of a Future to an ostream.
-// This function is found dynamically by googletest to print a Future object
-// in a test failure message.
-void PrintTo(const Future<void>& future, std::ostream* os);
+// This function is via ADL by Googletest to print a Future object in a test
```

Comment:
I think there is a missing verb in this sentence: "This function is via ADL...". Should it be "is *found*"?